scss/standalone/patterns_badge.scss

```diff
@@ -0,0 +1,4 @@
+@import '../vanilla';
```

Comment:
In the examples of badges we use other components as well, so we will have to add those here.

Otherwise only badges have their styling.
https://vanilla-framework-4497.demos.haus/docs/examples/standalone/patterns/badge/tabs
https://vanilla-framework-4497.demos.haus/docs/examples/standalone/patterns/badge/side-navigation
https://vanilla-framework-4497.demos.haus/docs/examples/standalone/patterns/badge/chips

So I guess we need to add tabs, chips and side navigation here.